private/dev/package.json

```diff
@@ -18,7 +18,7 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --clearScreen false",
```

Comment:
What is this unrelated change?

Reply:
just because i don't want to make a seaparate pr for it

Reply:
see commit https://github.com/transloadit/uppy/pull/4365/commits/d63a5aae01dde22db3684932b842f618283569db